lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -253,6 +258,28 @@ sub _route_path {
   return $path;
 }
 
+sub _security_action {
+  my ($self, $security_settings) = @_;
+  my $security_cb = $self->_security_cb;
+
+  return sub {
+    my $c = shift;
+    my @cc
+      = map { my ($name, $config) = %$_; ($security_cb->{$name}, $config); } @$security_settings;
+    my $wrapper;
+
+    $wrapper = sub {
+      my $c = shift;
+      my $cb = shift @cc || sub { die 'No security handler defined' };
```

Comment:
The fatal exception is too vague.
It should state something like:
"You haven't defined any 'Security Definitions Object'-implementations as a OpenAPI-plugin 'security'-parameter"

This way one immediately knows where to look for what is missing.
Otherwise one must dig through the whole codebase to figure out why this error is given.